frontend/src/app/containers/App.js

```diff
@@ -65,8 +65,16 @@ class App extends Component {
   }
 
   debounceSendToGA(pathname, type, dataIn) {
+    const { hasAddon } = this.props;
+
     if (this.lastPingPathname === pathname) { return; }
     this.lastPingPathname = pathname;
+    // Keep an environment variables to identify page state
+    // https://help.optimizely.com/Build_Campaigns_and_Experiments/AngularJS,_Backbone.js,_React,_and_Single_Page_Applications_in_Optimizely_Classic#1._Identify_states_in_your_Single_Page_App
```

Comment:
We currently are not using Optimizely X. Upgrading costs $, and is controlled by marketing. Don't think we'd be a driver for them to upgrade.